 CMakeLists.txt |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6caf4bf..7360edb 100755
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -800,7 +800,7 @@ ELSE(WIN32)
 		${${PROJECT_NAME}_BINARY_DIR}/env.sh
 	)
 ENDIF(WIN32)
-INSTALL(FILES ${ENV_SCRIPTS}
+INSTALL(PROGRAMS ${ENV_SCRIPTS}
 	DESTINATION ${${PROJECT_NAME}_INSTALL_SCRIPT_DIR})
 
 FILE(GLOB header_files ${${PROJECT_NAME}_SOURCE_DIR}/inc/*.*)
